tools/cli/go-whisk-cli/commands/qualifiedname.go

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commands
+
+import (
+    "errors"
+    "fmt"
+    "strings"
+    "../../go-whisk/whisk"
+    "../wski18n"
+)
+
+type QualifiedName struct {
```

Comment:
I like that all the qualified name functions are now part of the QualifiedName type. Having all the functionality for a type consolidated in one place make the code easier to follow. 👍